src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/NonGCStaticsNode.cs

```diff
@@ -19,10 +19,12 @@ internal class NonGCStaticsNode : ObjectNode, ISymbolNode
     {
         private MetadataType _type;
         private ISymbolNode _classConstructorContext;
+        private bool _hasLazyClassConstructor = false;
```

Comment:
Actually we don't need the _classConstructorContext anymore now we use a single symbol for the context and non-gc data. I'll remove.